config/env_sample.hjson

```diff
@@ -27,14 +27,14 @@
         "REPORT_ONLY": true,
         # You should set your domain as the DEFAULT_SRC
         "DEFAULT_SRC": ["'self'","example.edu"],
-        "SCRIPT_SRC": ["'self'", "'unsafe-inline'", "'unsafe-eval'", "www.google-analytics.com"],
-        "IMG_SRC": ["'self'", "data:", "www.google-analytics.com"],
+        "SCRIPT_SRC": ["'self'", "'unsafe-inline'", "*.googletagmanager.com"],
+        "IMG_SRC": ["'self'", "data:", "*.google-analytics.com", "*.googletagmanager.com"],
         "OBJECT_SRC": [],
         "MEDIA_SRC": [],
         # If you're embedding in Canvas you may need to include instructure.com as FRAME_SRC
         "FRAME_SRC": [],
         "FONT_SRC": ["'self'", "fonts.gstatic.com"],
-        "CONNECT_SRC": [],
+        "CONNECT_SRC": ["*.google-analytics.com", "*.analytics.google.com", "*.googletagmanager.com"],
         "STYLE_SRC": ["'self'", "'unsafe-inline'"],
```

Comment:
We may be able to remove `unsafe-inline` from `style-src` using the nonce added here.